code/modules/assembly/voice.dm

```diff
@@ -32,6 +32,10 @@
 	if(speaker == src)
 		return
 
+	// raw_message can contain multiple spaces between words etc which are not seen in chat due to HTML rendering
+	// this means if the teller records a message with e.g. double spaces or tabs, other people will not be able to trigger the sensor since they don't know how to perform the same combination
+	raw_message = htmlrendertext(raw_message)
```

Comment:
You should move this under listening check so it doesn't run the regex needlessly. Sorry didn't notice this before.

Reply:
Yeah I was considering that but that could create another issue where in a scenario, someone naughty says the triggering phrase but shoves in double spacing so the device won't get triggered, person number 2 could then assume that this phrase is fine and repeats it but does not know about the double spacing and will thus trigger it. To person number 3, it would have appeared that both said the exact same thing but it only triggered for one of them.

Bit of an edge-case scenario but still.

Reply:
Oh wait, i didn't notice the other branch actually does something with the raw message, my bad. All good.

Reply:
Thanks!